### DISCUSSION TITLE
Building Simulator Build through codemagic workflow for flutter project not available.

### DISCUSSION THREAD

**Discussion by sad-adnan** [codemagic-ci-cd/codemagic-docs#2572] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

i am trying to build and simulator build file (.app) file for my flutter project not the physical one, i found the doc for creating it for native ios projects  here (https://docs.codemagic.io/yaml-code-signing/ios-simulator-builds/) but i am already using workflow editor and there is no doc to try this one, can anyone help to export simulator build (.app) for the flutter project using workflow editor.

btw, i will need to upload the .app.zip file to https://appetize.io/upload where i can let others preview the app from their browser. but the current .app.zip says its a physical build, they need a simulator build. Thanks in advance for your help.

### Expected results

there should be a documantation or a way for building simulator build (.app) for flutter workflow editor.

### Actual results

cant find any way or documantation to build simulator build .app through workflow editor.

**Comment by himesh-cm** [comment]
Hey @sad-adnan, please consider moving to `yaml` for the above use case, as you need customisation.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
